src/dialog/dialog.tsx

```diff
@@ -50,23 +58,27 @@ class Dialog extends React.Component<Props, {}> {
                     data-qa={`dialog`}
                 >
                     <div className="dialog-content">
-                        <div className="dialog-content-header">
-                            <h3 className="dialog-title">{this.props.title}</h3>
-                        </div>
-                        <div className="dialog-content-body">{this.props.content}</div>
-                        <div className="dialog-content-footer">
-                            <Button
-                                id="cancel"
-                                text={this.props.btnCancelText}
-                                className="btn btn-secondary"
-                                onClick={this.props.onCancel}
-                            />
-                            <Button
-                                id="success"
-                                text={this.props.btnConfirmText}
-                                className="btn btn-primary"
-                                onClick={this.props.onConfirm}
-                            />
+                        <button id="cancel" className={'cancel'} onClick={this.props.onCancel}>
+                            <span className={`icon icon-close`} />
+                        </button>
+                        {this.props.type && (
+                            <div className="dialog-content-icon-wrapper" data-qa={`type-${this.props.type}`}>
+                                <span className={`icon icon-${this.props.type}`} />
```

Comment:
@MostafaBalata I'm a bit late, i know, but when type is not set this will result in `icon-undefined` right?